### DISCUSSION TITLE
Maintenance issue

### DISCUSSION THREAD

**Discussion by gund** [gund/eslint-plugin-deprecation#43] [General]
Hey folks!

I just wanted to address the maintenance issue that this plugin experienced and affected some (or many?) people.

I won't go into details of reasons but I would be happy to get a few more people in to help maintain this project so this situation does not happen again.

I would first extend my invite to some folks who have actively contributed PRs but you can also propose yourself in the comments.

So would any of you guys like to take up the maintenance role of this repo:
- @Delagen
- @gluxon
- @ecraig12345
- @kirkobyte

Please let me know if you are interested.
I'm looking for 2-3 people so there is always someone to merge a PR =)

Cheers!

**Comment by gluxon** [comment]
I appreciate this post @gund! Thanks for all the work you put into this project. ❤️

I'll wait a bit to see if others respond first and accept your invite if no on else has the time.

**Comment by delagen** [comment]
I can take apart, but i don't have solid experience in linting, but have enough in js and ts, so promise to not merge broken MR

**Comment by ecraig12345** [comment]
I'd be happy to help but need to clear it with my employer first (guessing it will be fine)

**Comment by gund** [comment]
Alright, thanks all above for your interest!
I will send you all invites soon so hopefully this plugin is well maintained going forward 👌
Cheers!

EDIT: The invites have been sent, please check your emails.
